### PR TITLE
Windows mingw

### PR DESCRIPTION
**Pull request by filimonov**

### CODE REVIEW

**Comment by filimonov** [comment]
ColumnsCase.Date2038 on i686 - real issue?

File: .github/workflows/windows_mingw.yml [hunk 1]
```diff
@@ -0,0 +1,86 @@
...
+jobs:
+  build:
+    runs-on: windows-latest
+
+    strategy:
+      fail-fast: false
+      matrix:
+        include:
+          - { sys: mingw64, env: x86_64 }
+          - { sys: mingw32, env: i686 }
```
Comment (Enmk) [inline .github/workflows/windows_mingw.yml:42]: Isn't `i686` obsolete and long forgotten arch of the Pentium-IV era? IMO, we shouldn't waste resources building\testing against that, unless there is a really strong reason.

Comment (filimonov) [reply .github/workflows/windows_mingw.yml:42]: ok for me. Just tried the whole matrix available  in mingw

**Review by Enmk — APPROVED** [review]
LGTM

**Pull request merged**